example/example-sequelize.js

```diff
@@ -0,0 +1,185 @@
+'use strict'
+
+// Custom Store using Sequelize ORM for PostgreSQL database
+
+// Sequelize Migration for "RateLimits" table
+//
+// module.exports = {
+//   up: (queryInterface, { TEXT, INTEGER, BIGINT }) => {
+//     return queryInterface.createTable(
+//       'RateLimits',
+//       {
+//         Route: {
+//           type: TEXT,
+//           allowNull: false
+//         },
+//         Source: {
+//           type: TEXT,
+//           allowNull: false,
+//           primaryKey: true
+//         },
+//         Count: {
+//           type: INTEGER,
+//           allowNull: false
+//         },
+//         TTL: {
+//           type: BIGINT,
+//           allowNull: false
+//         }
+//       },
+//       {
+//         freezeTableName: true,
+//         timestamps: false,
+//         uniqueKeys: {
+//           unique_tag: {
+//             customIndex: true,
+//             fields: ['Route', 'Source']
+//           }
+//         }
+//       }
+//     )
+//   },
+//   down: queryInterface => {
+//     return queryInterface.dropTable('RateLimits')
+//   }
+// }
+
+const fastify = require('fastify')()
+const Sequelize = require('sequelize')
+
+const databaseUri = 'postgres://username:password@localhost:5432/fastify-rate-limit-example'
+const sequelize = new Sequelize(databaseUri)
+// OR
+// const sequelize = new Sequelize('database', 'username', 'password');
+
+// Sequelize Model for "RateLimits" table
+//
+const RateLimits = sequelize.define(
+  'RateLimits',
+  {
+    Route: {
+      type: Sequelize.TEXT,
+      allowNull: false
+    },
+    Source: {
+      type: Sequelize.TEXT,
+      allowNull: false,
+      primaryKey: true
+    },
+    Count: {
+      type: Sequelize.INTEGER,
+      allowNull: false
+    },
+    TTL: {
+      type: Sequelize.BIGINT,
+      allowNull: false
+    }
+  },
+  {
+    freezeTableName: true,
+    timestamps: false,
+    indexes: [
+      {
+        unique: true,
+        fields: ['Route', 'Source']
+      }
+    ]
+  }
+)
+
+function RateLimiterStore (options) {
```

Comment:
I think that moving this class to a file would let the user understand better how to organize the source.

Moreover, this would avoid having a global `sequelize` object and it would be really more useful to users to adapt to their fastify instance

(I saw there is the same issue in the knex example and I think it should be improved as well - not an issue with this PR)